packages/remix-dev/cli/run.ts

```diff
@@ -376,7 +379,7 @@ export async function run(argv: string[] = process.argv.slice(2)) {
       if (hasInitScript) {
         if (installDeps) {
           console.log("💿 Running remix.init script");
-          await commands.init(projectDir);
+          await commands.init(projectDir, { deleteScript: true });
```

Comment:
This isn't really needed as this is the default

Reply:
Indeed not necessary, but I like the explicitness.